src/main/resources/application.properties

```diff
@@ -28,8 +28,10 @@ terra.common.stairway.forceCleanStart=false
 terra.common.stairway.migrateUpgrade=true
 terra.common.stairway.maxParallelFlights=120
 terra.common.stairway.clusterNameSuffix=stairwaycluster
+terra.common.stairway.tracing-enabled=false
```

Comment:
Where is this property used? I looked in `terra-common-library` and `stairway` but didn't see it mentioned in either place. 

Actually I don't see many of these stairway properties anywhere, so I must be looking in the wrong place. Do you know where they are?

Reply:
I got the properties from here: https://github.com/DataBiosphere/terra-resource-buffer/blob/master/src/test/resources/application-test.yml